src/metabase/models/card.clj

```diff
@@ -18,7 +18,8 @@
     :scalar
     :state
     :table
-    :timeseries})
+    :timeseries
+    :xkcd})
```

Comment:
i don't believe there is any reason to have this here.  i reverted this locally and everything works fine.